lib/data/datasources/remote/storage_service/firestore_service_impl.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter_master/data/datasources/remote/storage_service/firestore_service.dart';
+
+class FirestoreServiceImpl implements FirestoreService {
+  final FirebaseFirestore _firestore = FirebaseFirestore.instance;
+
+  @override
+  Future<List<Map<String, dynamic>>> getQuestions() async {
+    try {
+      final snapshot = await _firestore.collection('questions').get();
```

Comment:
StorageServiceFirebaseImpl name better and pass things like questions to consts, it is at least reused

Reply:
done